src/panel/pets.ts

```diff
@@ -221,7 +221,7 @@ abstract class BasePetType implements IPetType {
         this.currentState = resolveState(this.currentStateEnum, this);
 
         this._name = name;
-        this._speed = speed; // TODO #183 : Add a random modifier (+/- 30%) to this value.
+        this._speed = this.randomiseSpeed(speed);
```

Comment:
please change to the American English spelling of randomize.

Even though I'm British, this repository uses American English spelling!